packages/embed-examples/src/reactComponent.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="style.scss" />
+    <title>React Component example</title>
+    <script type="module" src="/reactComponent.jsx"></script>
```

Comment:
Should be at the end of body. To make it consistent with other examples.

Reply:
Fixed in https://github.com/genome-spy/genome-spy/pull/255/commits/a5144241529694463dd8ae9b0405b551b2e38228